packages/template-retail-react-app/app/pages/product-list/index.test.js

```diff
@@ -108,7 +122,6 @@ const server = setupMockServer(
 )
 
 beforeEach(() => {
-    window.history.pushState({}, 'ProductList', '/uk/en-GB/category/mens-clothing-jackets')
```

Comment:
We can leave this `pushState` by default for the majority of the tests.

Reply:
Since we are adding tests for both routes /search and /category/:id for PDP, having module-wise navigation to the category page for all tests does not make any sense for the one that needs to navigate to the search page.  I think it is better for the individual tests to handle the URL navigation instead of having a global one for all tests.

Reply:
I can see advantages for both approaches, less code vs more specific code. Either way works great in this case. 
Feel free to ignore the `window.history.pushState` comments @yunakim714.